packages/evm/README.md

```diff
@@ -94,4 +94,5 @@ by the Signer.
 regardless of the validity of individual action requests presented to the signer.
 
 Most incentives hard-cap each address to one claim, but `ERC20PeggedVariableCriteriaIncentive`
-does not, by design.
+and `ERC20VariableIncentive` do not, by design. `ERC1155Incentive` catalogs
+individual claims by transaction hash, as opposed to claimant.
```

Comment:
@Quazia I went through the incentives and called out all that don't track claim target.